### ISSUE TITLE
Click-through on Camera entities not working 

### ISSUE THREAD

**Issue by bagelos** [home-assistant/frontend#9550]
### The problem

I am having issues with the Foscam integration and or camera component of Core,  Could also be the Camera entity and picture card for displaying the camera feed.  

Previously, clicking through on a camera entity within a Pictire Entity Card, would bring up a window with a stream of the camera (in this case I have 6 x Foscam cameras).  

Since 2021.07 - the click-through window shows and it has some new features by the look of it, but the window is very small (vs 2021.06) and is black, e.g. no video, not even a static image.  

I am running Python 3.9 venv on ubuntu 20, latest everything inclunding ffmpeg, av libararies etc.  Also no issues with phyton av library.  

### What is version of Home Assistant Core has the issue?

core-2021.7.1

### What was the last working version of Home Assistant Core?

core-2021.6.6

### What type of installation are you running?

Home Assistant Core

### Integration causing the issue

Foscam

### Link to integration documentation on our website

https://www.home-assistant.io/integrations/foscam/

### Example YAML snippet

```yaml
Configure via GUI
```

### Anything in the logs that might be useful for us?

```txt
Will upload once I've done some more digging around.
```

**Comment by frenck** [comment]
The issue described sounds like a frontend issue, transferring the issue to the right repository.

**Comment by bramkragten** [comment]
Can you share a screenshot?

**Comment by bagelos** [comment]
Hi Bram, I plugged in another old camera I had and reset it to factory defaults and it seemed to work, although I had also automatically updated to core-2021.7.2 overnight.  I also checked default settings on old camera vs new and only difference was bitrate of the main stream, so I reduced this to 2M and it started working.  So i have set 2M bitrate on all cameras.  Thanks for your help/picking up the issue.  cheers George
